.github/workflows/create_tag_on_bump_merge.yml

```diff
@@ -1,32 +1,28 @@
 name: Create Tag Version on Bump Merge
 
 on:
-  push:
-    branches:
-      - master
+  pull_request:
+    types: [closed]
 
 jobs:
-  tag-version:
-    if: startsWith(github.event.head_commit.message, 'Merge pull request #') && contains(github.event.head_commit.message, 'version-bump-')
+  version_bump:
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'version-bump-2.221.0-')
```

Comment:
FYI I don't know if this was just one off for testing, but the version number ended up hardcoded here so all the subsequent runs skipped the CI since: [`actions/workflows/create_tag_on_bump_merge` runs](https://github.com/fastlane/fastlane/actions/workflows/create_tag_on_bump_merge.yml)

Reply:
BTW isn't the tag actually created by the release workflow at  [`fastlane create_github_release`](https://github.com/fastlane/fastlane/actions/runs/9557908294/job/26345741272#step:5:2216) step?